cli/internal/runsummary/format_json.go

```diff
@@ -55,12 +56,13 @@ func (rsm *Meta) normalize() {
 // This struct exists solely for the purpose of serializing to JSON and should not be
 // used anywhere else.
 type nonMonorepoRunSummary struct {
-	ID                ksuid.KSUID        `json:"id"`
-	Version           string             `json:"version"`
-	TurboVersion      string             `json:"turboVersion"`
-	GlobalHashSummary *GlobalHashSummary `json:"globalCacheInputs"`
-	Packages          []string           `json:"-"`
-	EnvMode           util.EnvMode       `json:"envMode"`
-	ExecutionSummary  *executionSummary  `json:"execution,omitempty"`
-	Tasks             []*TaskSummary     `json:"tasks"`
+	ID                ksuid.KSUID         `json:"id"`
+	Version           string              `json:"version"`
+	TurboVersion      string              `json:"turboVersion"`
+	GlobalHashSummary *GlobalHashSummary  `json:"globalCacheInputs"`
+	Packages          []string            `json:"-"`
+	Pipeline          fs.PristinePipeline `json:"pipeline"`
```

Comment:
We don't need this here

Reply:
Collapsing this conversation over into: https://github.com/vercel/turbo/pull/4545#discussion_r1178611355